lua/orgmode/org/indent.lua

```diff
@@ -25,7 +25,7 @@ local get_matches = ts_utils.memoize_by_buf_tick(function(bufnr)
       }
 
       if type == 'headline' then
-        opts.stars = vim.treesitter.query.get_node_text(node:field('stars')[1], bufnr):len()
+        opts.stars = vim.treesitter.get_node_text(node:field('stars')[1], bufnr):len()
```

Comment:
I think we need to use compat here and below in whole file

Reply:
Done